smoke-tests/spec/external_dns_helper.rb

```diff
@@ -40,16 +41,17 @@ def delete_A_record(zone_id, zone_name, domain_name)
   })
 end
 
-def delete_TXT_record(zone_id, zone_name)
+def delete_TXT_record(zone_id, zone_name, domain_name, namespace)
```

Comment:
Please change the method name to `delete_txt_record`, to follow ruby convention on method names